packages/components/src/components/select/select/select.tsx

```diff
@@ -128,20 +126,23 @@ export class Select {
   /** Adapts the select color depending on the theme. */
   @Prop() public theme?: Theme = 'light';
 
+  /** The id of a form element the select should be associated with. */
+  @Prop() public form?: string;
```

Comment:
How does the form association work now? Also an E2E test for his would be good (Same for `multi-select`)